src/commands.cc

```diff
@@ -2553,9 +2553,7 @@ const CommandDesc change_directory_cmd = {
              return { 0_byte, cursor_pos,
                       complete_filename(prefix,
                                         context.options()["ignored_files"].get<Regex>(),
-                                        cursor_pos, FilenameFlags::OnlyDirectories),
-                      Completions::Flags::Menu };
-        }),
+                                        cursor_pos, FilenameFlags::OnlyDirectories)};}),
```

Comment:
@mawww  and this one here I don't like it. Do you want to hear my honest opinion? (how the saying goes).  We'd be better off by wait it out I think. But since when has become an impediment to use the <backspace> freely? Heck. The completions have to work and have to take me to the right dir no matter how many times I press the backspace key or any other key for that matter.

Reply:
this is an unrelated change, don't include it here.
I have a different fix prepared, will submit it soon.